server/routes/group/add-users-to-group.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const { HttpError } = require('../../utils/errors');
+const { Group } = require('../../models');
+const { authenticateToken } = require('../../utils/token-helpers');
+
+const router = express.Router();
+router.post('/:id/user', authenticateToken, async (req, res) => {
+    const { data } = req.body;
+    const { id } = req.user;
+    if (!Array.isArray(data)) {
+        return res.status(400).json({ message: 'Invalid Payload' });
+    }
+
+    try {
+        await Group.addUsers(data, req.params.id, id);
```

Comment:
```suggestion
        await Group.addUsers(data, group_id, id);
```
This looks more readable.